src/main/java/uk/gov/pay/adminusers/model/Invite.java

```diff
@@ -38,6 +43,16 @@ public String getTelephoneNumber() {
         return telephoneNumber;
     }
 
+    @JsonProperty("disabled")
+    public Boolean getDisabled() {
```

Comment:
shouldn't boolean properties normally named as `is`.. For e.g. `isDisabled`. (Have a look at the other examples. User.java)